### ISSUE TITLE
Fix cell highlights in entry view when using KeePassXC themes

### ISSUE THREAD

**Issue by ogmios-voice** [keepassxreboot/keepassxc#7330]
## Overview
The actual grid behaviour for `Enter` (key) depends on the **selected column.** What makes things a bit more difficult: the focus in the grid (thus the selected column) is visible only while using the Classic theme.

## Steps to Reproduce
1. Optional: switch to Classic theme
2. Open any group w/ at least one entry or do a search w/ at least one result
3. Click (only once) on the different columns in the row
4. Press `Enter`

Note: when the grid is rebuilt (e.g. a new group is opened, or a new search is triggered) the default selected column will be the _Title_.

## Expected Behavior
Two possible solutions:
1. Regardless of the selected column, `Enter` should always do the same -- this would however contradict the very useful behaviour on double click
2. Make sure the focus on the selected column is always visible, and allow changing it from the keyboard as well (e.g. `Ctrl + Left/Right`)
3. Or maybe both + a config option for the user

Nice to have: selecting the columns in the popup opened on right-clicking the grid header should allow multiple selections w/o close.

## Actual Behavior
Enter will trigger the same action as double-click. (Yet the user might not know which column is actually selected, thus what to expect.)

By columns:
* _Group_: open given group
* _Has attachments, Title, Notes, Expires, Created, Modified, Accessed, Attachments, Size_: edit entry
* _Has TOTP_: setup TOTP (dialog)
* _Username, Password_: copy to clipboard
* _URL_: open URL in browser

## Context
KeePassXC - Version 2.6.6
Revision: 9c108b9
Qt 5.15.2
Operating system: Windows 10 Version 2009
CPU architecture: x86_64
Kernel: winnt 10.0.19043

**Comment by droidmonkey** [comment]
We already added a config option to disable the double click behavior on cells, which is now disabled by default. It is consistent for Enter to do the same thing as Double Click, they both mean "activate". If you always want to be sure you are going to edit an entry, use the dedicated shortcut Ctrl+E.

However, I do agree the cell highlighting needs to be better in our custom theme.
